c2_components/include/mfx_c2_decoder_component.h

```diff
@@ -281,6 +281,7 @@ class MfxC2DecoderComponent : public MfxC2Component
     std::shared_ptr<C2StreamColorAspectsTuning::output> m_defaultColorAspects;
     std::shared_ptr<C2StreamColorAspectsInfo::input> m_codedColorAspects;
     std::shared_ptr<C2StreamColorAspectsInfo::output> m_colorAspects;
+    std::shared_ptr<C2StreamUsageTuning::output> m_outputUsage;
```

Comment:
We already have consumer usage declared here.
http://inlubt0232.iind.intel.com:8080/source/xref/CELADON/vendor/intel/mediasdk_c2/c2_components/include/mfx_c2_decoder_component.h?r=16901798#246

Please refine this patch.

Thanks,
Tianmi

Reply:
the old one is removed